### ISSUE TITLE
Add AA Torque

### ISSUE THREAD

**Issue by agronick** [shmykelsa/AAAD#265]
With Performance Monitor abandoned I started rewriting it to support any car. You can see the releases [here](https://github.com/agronick/aa-torque/releases) and a video [here](https://www.reddit.com/r/AndroidAuto/comments/16wmlaz/im_working_on_a_torque_plugin_for_android_auto/). It would be great if you could add it.

**Comment by aross6** [comment]
Agreed. AA Torque would be a GREAT addition to AAAD, especially with Performance Monitor being abandoned. Please add!

**Comment by Gullzway** [comment]
You can add this to Android Auto yourself, just use KingInstaller.

**Comment by shmykelsa** [comment]
Added as of 2.0
